pyvodb/load.py

```diff
@@ -53,21 +53,30 @@ def dict_from_directory(directory, root, ignored_files=()):
     return data
 
 
+def load_meta_file(directory):
+        metafile = os.path.join(directory, 'meta.yaml')
+        with open(metafile) as f:
+            info = yaml.load(f, Loader=YAML_SAFE_LOADER)
+        info['ignored_files'].append('meta.yaml')
```

Comment:
This removes `meta` from the result of `load_from_directory`. I'd rather have it available.